### PR TITLE
Update intercom_messages_to_bigquery.py

### PR DESCRIPTION
**Pull request by malcolmtivelius**

### CODE REVIEW

**Comment by malcolmtivelius** [comment]
This updates the DAG to Airflow 2.0 standard

File: anyfin-platform/platform-composer/dags/intercom_to_bigquery/intercom_messages_to_bigquery.py [hunk 1]
```diff
@@ -1,35 +1,31 @@
...
-	'owner': 'ds-anyfin',
+	'owner': 'de-anyfin',
 	'depends_on_past': False,
 	'start_date': datetime(2020, 9, 8),
 	'retries': 2,
 	'retry_delay': timedelta(minutes=10),
-	'email_on_failure': True,
-	'email_on_retry': False,
-	'email': Variable.get('de_email', 'data-engineering@anyfin.com')
+	#'email_on_failure': True,
```
Comment (liza-mazurova) [inline anyfin-platform/platform-composer/dags/intercom_to_bigquery/intercom_messages_to_bigquery.py:26]: maybe we delete it instead of comment out?

Comment (malcolmtivelius) [reply anyfin-platform/platform-composer/dags/intercom_to_bigquery/intercom_messages_to_bigquery.py:26]: Its just so I don't forget to add the Slack thing instead

**Review by liza-mazurova — APPROVED** [review]
LGTM

**Pull request merged**